endpoints.go

```diff
@@ -63,8 +63,9 @@ var (
 	EndpointUser               = func(uID string) string { return EndpointUsers + uID }
 	EndpointUserAvatar         = func(uID, aID string) string { return EndpointCDNAvatars + uID + "/" + aID + ".png" }
 	EndpointUserAvatarAnimated = func(uID, aID string) string { return EndpointCDNAvatars + uID + "/" + aID + ".gif" }
-	EndpointDefaultUserAvatar  = func(uDiscriminator int) string {
-		return EndpointCDN + "embed/avatars/" + strconv.Itoa(uDiscriminator%5) + ".png"
+	EndpointDefaultUserAvatar  = func(uDiscriminator string) string {
+		uDiscriminatorInt, _ := strconv.Atoi(uDiscriminator)
+		return EndpointCDN + "embed/avatars/" + strconv.Itoa(uDiscriminatorInt%5) + ".png"
```

Comment:
Oh you need the int for modulus, guh.

I regret asking you for the previous change now, sorry!

But now that we need to do the Atoi, can you check the error in user.go and return "" if it fails?

Reply:
if it fails it returns 0, the final url will be https://cdn.discordapp.com/embed/avatars/0.png, which is a valid url. I think that might be better to return instead of just ""?